content/_help/verify-your-identity/how-to-verify-your-identity._en.md

```diff
@@ -21,7 +21,7 @@ If you are under 18 years of age, you may not be able to verify your identity fo
 1. Your State-Issued ID. You can upload a photo of your ID by phone or by computer. [Don’t have a state issued ID?](/help/verify-your-identity/accepted-state-issued-identification/) 
 1. Social Security number
 1. Your phone number 
-    * If we can't verify your phone number, you can verify by mail instead which takes approximately 3-7 business days. 
+    * If we can't verify your phone number, you can [Verify by mail](/help/verify-your-identity/verify-your-address-by-mail/) instead. This takes 5 to 10 days.
```

Comment:
Are we intentionally capitalizing "Verify" here? We don't in the other languages.

Reply:
Thanks for the catch. Fixed. I wasn't sure how we were handling mid-sentence links at first.